src/components/SecondaryPanes/Frames/index.js

```diff
@@ -176,6 +182,7 @@ class Frames extends Component {
     return dom.div(
       { className: "pane frames" },
       this.renderFrames(frames),
+      WhyPaused(),
```

Comment:
now that WhyPaused is a child of Frames, it would be nice to pass pauseInfo down `WhyPause({ pauseInfo })`. This means we no longer need to have the connect logic in WhyPaused too :)